.travis.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     os: linux
     arch: ppc64le
     jdk: openjdk11
-    script: mvn install -DskipTests -Dmaven.javadoc.skip=true
+    script: mvn install -Dmaven.javadoc.skip=true
```

Comment:
Does it still need to skip the JavaDoc?

Reply:
Yes, was facing some issues with javadoc.skip flag enabled. Will revert once it is resolved.